doc/scripts/extract_content.py

```diff
@@ -89,11 +90,22 @@ def src_deps(zephyr_base, src_file, dest):
         if not m:
             continue
 
-        dep_rel = m.group('dep_rel')  # relative to src_dir
+        dep_rel = m.group('dep_rel')  # relative to src_dir or absolute
+        # Try a relative path first
         dep_src = path.abspath(path.join(src_dir, dep_rel))
+        # Not a relative path, check if it's absolute if we have been
+        # provided with a sphinx source directory root
+        if path.isabs(dep_rel) and src_root:
+            # Make it really relative
+            dep_rel = '.' + dep_rel
+            dep_src = path.abspath(path.join(src_root, dep_rel))
+            if path.isfile(dep_src):
+                # File found, but no need to copy it since it's part
+                # of Sphinx's top-level source directory
+                continue
         if not path.isfile(dep_src):
             print("File not found:", dep_src, "\n  referenced by:",
-                  src_file, file=sys.stderr)
+                   src_file, file=sys.stderr)
```

Comment:
I meant "why change the whitespace at all?"; it was fine before.

Reply:
ops, made a mistake, fixed!